src/components/Button/Button.css

```diff
@@ -1,9 +1,14 @@
 button {
   height: 30px;
-  width: 60px;
-  background-color: blue;
+  width: 100px;
+  background-color: #245abc;
   color: white;
-  font-size: 1.05rem;
-  margin-top: 10px;
+  border-radius: 2500rem;
+  border-style: solid;
   cursor: pointer;
+  display: inline-flex;
+  font-weight: 750;
+  align-items: center;
```

Comment:
I don't think we need the `align-items` or `justify-content` since we aren't not doing anything different than the parent flexbox.